opendocdb-cts/internal/mongosh/fixture_test.go

```diff
@@ -162,9 +178,75 @@ func TestConvertFixtures(t *testing.T) { //nolint:revive // exceeds number of li
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			actual, err := ConvertFixtures(tc.fixtures)
+			if tc.skip != "" {
+				t.Skip(tc.skip)
+			}
+
+			ctx := context.Background()
+			l := slogt.New(t)
+			dbName := "test"
```

Comment:
Why not use unique database for each test case like `t.Name()`? Then this test and test case can run in parallel and `dropDatabase` shouldn't cause issue in https://github.com/OpenDocDB/cts/pull/31/files#diff-63305a05443a3813daf63389a6978f98a124d4e5a9ab4686ff65b64d9014b814R213-R216